ibm_cloud_sdk_core/authenticators/cp4d_authenticator.py

```diff
@@ -59,14 +59,15 @@ def __init__(self,
                  apikey: str = None,
                  disable_ssl_verification: bool = False,
                  headers: Optional[Dict[str, str]] = None,
-                 proxies: Optional[Dict[str, str]] = None) -> None:
+                 proxies: Optional[Dict[str, str]] = None,
+                 verify: Optional[str] = None) -> None:
```

Comment:
Please also document the `verify` parameter in the docstrings above, similar to headers, proxies, etc.